frontend/packages/dev-console/integration-tests/support/step-definitions/web-terminal/web-terminal-adminuser.ts

```diff
@@ -0,0 +1,35 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { nav } from '@console/cypress-integration-tests/views/nav';
+import { switchPerspective } from '../../constants';
+import { webTerminalPO } from '../../pageObjects/web-terminal-po';
+import { perspective } from '../../pages';
+import { addTerminals, closeTerminal } from '../../pages/functions/addTerminalTabs';
+import { checkTerminalIcon } from '../../pages/functions/checkTerminalIcon';
+
+Given('user has logged in as admin user', () => {
+  perspective.switchTo(switchPerspective.Administrator);
+  nav.sidenav.switcher.shouldHaveText(switchPerspective.Administrator);
+});
+
+Given('user can see terminal icon on masthead', () => {
+  checkTerminalIcon();
+  cy.get(webTerminalPO.webTerminalIcon).should('be.visible');
+});
+
+When('user clicks on the Web Terminal icon on the Masthead', () => {
+  cy.get(webTerminalPO.webTerminalIcon).click();
+});
+
+Then('user opens {string} additional web terminal tabs', (n: string) => {
```

Comment:
this would be much easier to use a number parameter instead of a string, just use `{int}` for the expression and define `n` as `number`